lib/engines/content_block_manager/features/create_object.feature

```diff
@@ -20,8 +20,8 @@ Feature: Create a content object
     Then I should see a form for the schema
     And I should see a back link to the select schema page
     When I complete the form with the following fields:
-      | title            | email_address   | department | organisation |
-      | my email address | foo@example.com | Somewhere  | Ministry of Example |
+      | title            | email_address   | department | organisation        | instructions_to_publishers |
+      | my email address | foo@example.com | Somewhere  | Ministry of Example | this is important  |
```

Comment:
Wondering if there should be more test coverage on feature/integration level, as I've only added the field into the `create` spec... as it's an optional field that will give us coverage in `edit` feature spec that the user doesn't have to add it, but maybe should also add it to the edit feature test?

Reply:
done!